spec/Modules.tex

```diff
@@ -220,40 +226,57 @@ \section{Access of Module Contents}
 depending on the visibility of the module
 itself~(\rsec{Visibility_Of_A_Module}) and the visibility of each
 individual symbol~(\rsec{Visibility_Of_Symbols}).  This can be done
-via explicit naming~(\rsec{Explicit_Naming}) or the use
-statement~(\rsec{Using_Modules}).
+via the use statement~(\rsec{Using_Modules}) or qualified
+naming~(\rsec{Explicit_Naming}).
 
 \subsection{Visibility Of A Module}
 \label{Visibility_Of_A_Module}
 \index{modules!access}
 
-A module defined at file scope is visible anywhere. The visibility of a nested
-module is subject to the rules of~\rsec{Visibility_Of_Symbols}. There,
-the nested module is considered a "symbol defined at the top level
-scope" of its outer module.
+A top-level is available for use~(\rsec{Using_Modules}) anywhere. The
+visibility of a nested module is subject to the rules
+of~\rsec{Visibility_Of_Symbols}, where the nested module is considered
+a "module-scope symbol" of its outer module.
 
 \subsection{Visibility Of A Module's Symbols}
 \label{Visibility_Of_Symbols}
 \index{modules!access}
 
-A symbol defined at the top level scope of a module is \emph{visible}
-from outside the module when the \sntx{privacy-specifier} of its
-definition is \chpl{public} or is omitted (i.e. by default). When a
-symbol defined at the top level scope of a module is declared
+A symbol defined at module scope is \emph{visible} from outside the
+module when the \sntx{privacy-specifier} of its definition
+is \chpl{public} or is omitted (i.e. by default). When a module-scope
+symbol is declared
 \chpl{private}, it is not visible outside of that module. A
 symbol's visibility inside its module is controlled by normal lexical
-scoping and is not affected by its \sntx{privacy-specifier}.  A
-module's visible symbols are accessible via explicit
-naming~(\rsec{Explicit_Naming}) or the use
-statement~(\rsec{Using_Modules}) only where the module's symbol is
-visible~(\rsec{Visibility_Of_A_Module}).
+scoping and is not affected by its \sntx{privacy-specifier}.  When a
+module's symbol is visible~(\rsec{Visibility_Of_A_Module}), the
+symbols it contains are accessible via module's visible symbols are
+accessible via the use statement~(\rsec{Using_Modules}) or qualified
```

Comment:
I think this should say "When a module's symbol is visible, the visible symbols it contains are accessible via the use statement or qualified naming"

Reply:
Oh good lord.  I should really get my brain cleaned.  And people wonder why I hate working on documentation...